PermanentOrg/web-app
# Per 9569 redesign user journey

This pr adds the new user journey to the application.

Steps to test: 
Same as before, make sure that isGlam is set to true in your localStorage, then create a new account

Additionally, the new square component can be tested in storybook.
